test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -1505,10 +1554,15 @@
                         (deliver received-cmd? true)
                         @go-ahead-and-execute
                         (apply real-replace! args))]
-          (enqueue-command (command-names :replace-facts) 4
-                           {:environment "DEV" :certname "foo.local"
-                            :values {:foo "foo"}
-                            :producer_timestamp (to-string (now))})
+          (enqueue-command (command-names :replace-facts)
+                           4
+                           "foo.local"
+                           (-> {:environment "DEV" :certname "foo.local"
```

Comment:
This should be updated to the test store-command function, it'll coerce to a stream automatically